cerbero/packages/osx/packager.py

```diff
@@ -124,6 +124,8 @@ def pack(self, output_dir, devel=True, force=False, keep_temp=False, split=True,
 
         self.install_dir = install_dir or self.package.get_install_dir()
         self.version = version or self.package.version
+        if not sdk_version and hasattr(self.package, 'sdk_version'):
```

Comment:
sdk_version is passed as a parameter because OSXPackager works with a cerbero.packages.package.Package and this class does not have the sdk_version field, only MetaPackage does (https://github.com/fluendo/cerbero/blob/aef435a8b1d5c65b376d445ff846efd97eb8b7f7/cerbero/packages/package.py#L428). So this is checking a condition that would never happen

Reply:
In fact, sdk_version is used by fluendo-sdk (which is a package) and fluendo-sdk-installer (which is a meta-package). The issue comes from the initial conception of fluendo-sdk package. The result is that if you build fluendo-sdk package you will have a package installing itself in 1.1.0.xxx (which is the package version), whereas if you install fluendo-sdk-installer package it will install into 1.1 (which is the sdk_version).

Exactly the same files are installed, so it sounds quite incoherent. This is the meaning of this patch, if the package has a sdk_version field ALTHOUGHT it is NOT a meta-package, I read this field.

From the fluendo-sdk package point of view it seems more coherent and it doesn't have impact on other packages, but if you want I can remove it. Then everybody will need to be aware that UNLIKE all other platforms, fluendo-sdk and fluendo-sdk-installer produce different installation paths on Mac.

Reply:
Got it, it makes sense :)